src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,67 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        final BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        System.out.println("\n *** CREATE AND READ FUNCTIONS TEST - create and findById *** \n");
+
+        // CREATE
+        Book book1 = new Book();
+        book1.setTitle("Biały zamek");
+        book1.setPrice(BigDecimal.valueOf(20));
+
+        Book book2 = new Book();
+        book2.setTitle("Biały zamek");
+        book2.setPrice(BigDecimal.valueOf(20));
+
+        Book book3 = new Book();
+        book3.setTitle("Biały zamek");
+        book3.setPrice(BigDecimal.valueOf(20));
+
+        bookDao.create(book1);
+        bookDao.create(book2);
+        bookDao.create(book3);
+
+        // READ
+        System.out.println(bookDao.findById(1L));
+        System.out.println(bookDao.findById(2L));
+        System.out.println(bookDao.findById(3L));
+
+        System.out.println("\n *** UPDATE AND READ FUNCTIONS TEST - update and findAll *** \n");
+
+        // UPDATE
+        book1 = bookDao.findById(1L).get();
```

Comment:
The method `findById` returns an `Optional`. It's safer to check if the value is present before calling `get()` to avoid `NoSuchElementException`. Consider using `ifPresent` or `orElseThrow` for better handling.